src/com/mendix/widget/carousel/components/CarouselItem.ts

```diff
@@ -1,3 +1,4 @@
+import * as classnames from "classnames";
```

Comment:
Import it as `classNames` (from https://github.com/JedWatson/classnames). It's what the author recommends, so let's stick to that.